projen/projen
# feat: cdk8s-app-ts

REF: #694

Steps to test :
```
1. yarn projen new cdk8s-app-ts
2. yarn build
```
TODO:

- [ ] Sample code for tests

---

Extending from TypeScriptApp. Release and Build workflows have synth included.

Might have forgotten a few things, feel free to add them in the comments below : )

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
